packages/dds/merge-tree/src/test/client.spec.ts

```diff
@@ -207,6 +208,406 @@ describe("TestClient", () => {
 		});
 	});
 
+	describe(".searchForMarker", () => {
```

Comment:
not a big deal, but it could be nice to move all these new test to their own file, it makes them a bit easier to run if someone works in this space